asr-python-advanced-tao-ngram-deployment.ipynb

```diff
@@ -0,0 +1,497 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Syntax: riva-deploy -f dir-for-rmir/model.rmir:key output-dir-for-repository\n",
+    "! docker run --rm --gpus 0 -v $MODEL_LOC:/data $RIVA_SM_CONTAINER -- \\\n",
+    "            riva-deploy -f  /data/asr_riva_ngram_lm.rmir:$KEY /data/models/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## Start the Riva Server\n",
+    "After the model repository is generated, we are ready to start the Riva server. First, download the Riva Quick Start resource from NGC. \n",
+    "Set the path to the directory here:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the Riva Quick Start directory\n",
+    "RIVA_DIR = \"<Path to the uncompressed folder downloaded from quickstart(include the folder name)>\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we modify the `config.sh` file to enable relevant Riva services (n-gram language model), provide the encryption key, and path to the model repository (`riva_model_loc`) generated in the previous step among other configurations. \n",
+    "\n",
+    "For example, if above the model repository is generated at `$MODEL_LOC/models`, then you can specify `riva_model_loc` as the same directory as `MODEL_LOC`. <br>\n",
+    "\n",
+    "Pretrained versions of models specified in `models_asr/nlp/tts` are fetched from NGC. Since we are using our custom model, we can comment it in `models_asr` (and any others that are not relevant to your use case). <br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### config.sh snippet\n",
+    "```\n",
+    "# Enable or Disable Riva Services \n",
+    "service_enabled_asr=true                                                      ## MAKE CHANGES HERE\n",
+    "service_enabled_nlp=false                                                      ## MAKE CHANGES HERE\n",
+    "service_enabled_tts=false                                                     ## MAKE CHANGES HERE\n",
+    "\n",
+    "# Specify one or more GPUs to use\n",
+    "# specifying more than one GPU is currently an experimental feature, and may result in undefined behaviours.\n",
+    "gpus_to_use=\"device=0\"\n",
+    "\n",
+    "# Specify the encryption key to use to deploy models\n",
+    "MODEL_DEPLOY_KEY=\"tlt_encode\"                                                  ## MAKE CHANGES HERE\n",
+    "\n",
+    "# Locations to use for storing models artifacts\n",
+    "#\n",
+    "# If an absolute path is specified, the data will be written to that location\n",
+    "# Otherwise, a docker volume will be used (default).\n",
```

Comment:
it should always be Docker not docker - we've been called out on this in the past

Reply:
OK, thanks. This should be fixed in the CitriNet deployment notebook as well.